src/components/productView/__tests__/productView.test.js

```diff
@@ -8,7 +8,7 @@ describe('ProductView Component', () => {
       productConfig: {},
       routeDetail: {
         pathParameter: 'lorem ipsum',
-        productParameter: 'dolor sit'
+        viewParameter: 'dolor sit'
```

Comment:
required:
be helpful to include the prop that produces the productLabel since the fix for this revolves around it